dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -270,6 +266,10 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
     try {
       DeviceGuard g(device_id_);
 
+#if NVML_ENABLED
+      nvml::Shutdown();
```

Comment:
We have altered the semantics of shutdown WRT plain nvmlShutdown, which makes this kind of behaviour incorrect.
Please either revert or change:
```C++
inline void Shutdown() {
  std::lock_guard<std::mutex> lock(Mutex());
  if (!nvmlIsInitialized()) {
    return;
  }
  CUDA_CALL(nvmlShutdown());
}
```
to
```C++
inline void Shutdown() {
  CUDA_CALL(nvmlShutdown());
}
```